com.openmusicplayer.OMP.json

```diff
@@ -0,0 +1,257 @@
+{
+
+  "app-id": "com.openmusicplayer.OMP",
+
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
+
+  "sdk": "org.gnome.Sdk",
+
+
+  "finish-args":
+  [
+
+    "--socket=x11",
```

Comment:
For X11, you probably want `--share=ipc` as well.
And as it is a gtkmm application, it probably supports Wayland.

See also:
http://docs.flatpak.org/en/latest/sandbox-permissions.html#standard-permissions

Reply:
People interested OMP have asked for a Flatpak. I do consider it fairly stable at this point. The alpha quality is because a large amount of features I have planned are unimplemented and there's a few minor bugs I need to work out, but nothing that causes an abort or segfault that I know of.